src/utils/ping.ts

```diff
@@ -22,14 +22,18 @@
  * SOFTWARE.                                                                      *
  **********************************************************************************/
 
-const ping = require('ping')
+import ping from 'ping'
 
-const PING_TIMEOUT_S = 10 // 10 seconds
+const PING_TIMEOUT_S = 10 // 10 seconds timeout
 /**
  * sendPing() sends a ping to an address/host
  * @param host is a string
- * @returns
+ * @returns result of ping.promise.probe()
  */
 export async function sendPing(host: string) {
-  return ping.promise.probe(host, { timeout: PING_TIMEOUT_S })
+  const cleanUrl: any = host.replace(/^https?:\/\//, '') // sanitize url
+  return ping.promise.probe(cleanUrl, { timeout: PING_TIMEOUT_S })
```

Comment:
should timeout be configurable here?

either the default `response-time-greater-than-2-s` or from the config file

Reply:
hmm i dont think it should be a user configurable setting, don't see a need for it. It's an dev setting, set here so its not too long.